civihr_default_theme/scss/pages/_my-leave.scss

```diff
@@ -0,0 +1,10 @@
+.page-myleave {
```

Comment:
Since you decided to keep `#my-leave-block`, I think there's no use using `.page-myleave` as the namespace of this scss partial, you can use `#my-leave-block` directly:
```scss
#my-leave-block {
  padding: 0 15px;

  .button-container {
    margin-bottom: 40px;
  }
}
```